Makefile

```diff
@@ -21,7 +21,7 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.58.0
+PIPELINE_VERSION = pke-nodepool-update
```

Comment:
**Required**: this **MUST** be changed to the final pipeline version tag before merging. Also generation should be rerun, because there is a generated file with the version.

Reply:
fixed with the new 0.59.0 tag